bin/envconfig

```diff
@@ -164,6 +164,11 @@ export LSST_CONDA_ENV_NAME
 # shellcheck disable=SC1090
 . "$LSSTSW/eups/current/bin/setups.$SUFFIX"
 
+# definition EUPS_PATH depending on the environment:
+# including the last 7 characters of the environment name
+export EUPS_PATH="${LSSTSW}/stack${LSST_CONDA_ENV_NAME: -7}"
```

Comment:
Can't you just reuse `${LSST_SPLENV_REF}` here rather than computing it?  And please add a `-`.

Reply:
Sorry, I miss this point. envconfig will deploy whatever environment, and, from the way the envnames are formatted, the LSSP_SPLENV_REF is the last part of the name. I can't use the $LSST_SPLENV_REF here since, this is fixed to the default one, but envconfig shall be able to activate whatever environment, and assign the proper EUPS_PATH, not only the default one.